### DISCUSSION TITLE
Allow sorting Dumps window top to bottom

### DISCUSSION THREAD

**Discussion by marcusmoore** [beyondcode/herd-community#756] [Ideas]
It would be great to be able to optionally sort the Dumps window so new entries appear at the bottom of the list instead of the top. 

That would bring it in line with how Console.app or tailing from the terminal displays new information so dumping multiple items in a row allows you to read the code and dumps in the same order instead of having to read code top to bottom and the Dumps window bottom to top.

**Comment by mpociot** [comment]
Yup that's coming in the next update.

**Reply by marcusmoore** [reply]
Awesome! Thanks for the amazing work!
